### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by momotsow**
- Added a contact form 
- Added social media icon links on the footer

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -169,5 +169,50 @@ <h4 class="profile-tittle">Skills</h4>
...
+                <a href="#"><img class="resume-img" src="./images/ic-link-teal.svg" alt="ink teal"></a>
+            </div>
+            <div class="social-media">
+                <a href="#"><img class="social-link" src="./images/ic-stackoverflow.svg" alt=""></a>
+                <a href="#"><img class="social-link" src="./images/ic-github.svg" alt=""></a>
+                <a href="#"><img class="social-link" src="./images/ic-twitter.svg" alt=""></a>
+                <a href="#"><img class="social-link" src="./images/ic-linkedin.svg" alt=""></a>
+                <a href="#"><img class="social-link" src="./images/ic-medium.svg" alt=""></a>
+                <a href="#"><img class="social-link" src="./images/ic-angellist.png" alt=""></a>
+            </div>
```
Comment (mckent05) [inline index.html]: - For semantic purposes and accessibility, I'd suggest you wrap your links in a `ul` tag with each link in a `li` tag. Please try to fix this 🙏.

File: index.html [hunk 2]
```diff
@@ -169,5 +169,50 @@ <h4 class="profile-tittle">Skills</h4>
...
+
+    <section class="form-section">
+        <div class="header-img"></div>
+        <div class="form-container container">
+            <h1 class="tittle">Get started</h1>
+            <p class="para para2">
+                Lorem ipsum dolor sit amet, consectetur adipiscing
+                elit, sed do eiusmod tempor incididunt ut labore
+                et dolore magna.
+            </p>
```
Comment (mckent05) [inline index.html]: - To improve your site's accessibility, I'd advise you not to have more than one `h1` tag in your HTML code. Please kindly fix this 👍

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @momotsow, @Cisco-the-wayword, 

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters are passing correctly ✔️ ✔️ 
- Your portfolio is looking good and gradually starting to come together 👏 
- Descriptive PR summary and title 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by christianbiring1 — APPROVED** [review]
Hi @momotsow , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉
imageswelljob

## Highlights
- [ ] All linters checks are green
- [ ] Descriptive PR `title`
- [ ] Descriptive PR `summary`
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**